lib/cli/src/commands/app/mod.rs

```diff
@@ -117,7 +117,7 @@ pub async fn deploy_app_verbose(
     client: &WasmerClient,
     opts: DeployAppOpts<'_>,
 ) -> Result<(DeployApp, DeployAppVersion), anyhow::Error> {
-    let owner = &opts.owner;
+    let owner = &opts.owner.or_else(|| opts.app.owner.clone());
```

Comment:
While we are at it, let's also update the `app create` command to add the `owner` field.

Reply:
Done